### DISCUSSION TITLE
I can no longer get the Github Education Pack confirmed on my account

### DISCUSSION THREAD

**Discussion by artysan-code** [community/community#126637] [GitHub Education]
### Hi everyone,

I'm trying to figure out how to solve this problem that has been bothering me for almost 2 years.

in November 2022 I registered the Github Education Pack for my first time and used it for at least 6 months without any problems, completely satisfied.

After those 6 months I was asked to provide the proof again, but the server (I assume) received two requests at the same time (I think even multiple since the next day I received 9 verification notifications even though I never made them). From then on I was told that my email ******.*****@students.uniroma2.eu could not be accepted because it had already been used.

Too bad that my account no longer had Github Education activated and no one could be using this Pack. I tried for months in every way and made several posts but no possibility of solving it. Furthermore, I have now recreated a second account (first rgoth-main now sleepy-goth) with the exact same emails (so I no longer have access to the account associated with GitHub Education before as it was deleted).

I don't know what to do at all and I would like to take advantage of this Pack to be able to study as best as possible for my degree in Computer Science. I ask for help for anyone who can take the time to give me a hand, I thank you infinitely in advance.

**Comment by artysan-code** [comment]
Small addition, it tells me that the account has already been used for the rgoth-main account now and that account is deleted.


Furthermore, it gave the exact same message even when I used the rgoth-main account (senseless as it tells me that I cannot apply the discount to my account, because it is already associated with my account??)

**Comment by davevad93** [comment]
Hi @sleepy-goth ,

That's an odd issue indeed, I don't know why it's happening if you no longer have access to GitHub Education and you're 100% sure that no one else is using your educational email. The only thing I can recommend you to do is to contact support by opening a ticket [here](https://support.github.com/contact/account), I don't know what could be the proper category for this, so select `Other` and describe in detail your problem.

**Reply by artysan-code** [reply]
Thank you so much, now I'll try to contact them even if the tool rightly thinks every time that I'm creating a ticket for a problem that can be solved normally. I'll update as soon as they reply

**Reply by davevad93** [reply]
You're welcome man, I hope they'll reply soon. Notte!
